### ISSUE TITLE
[PL][Deadmines][NPC] Helix Gearbreaker

### ISSUE THREAD

**Issue by RastaTinu** [Atlantiss/BugTracker#8116]
Dungeon: http://www.wowhead.com/zone=1581/the-deadmines#drops
NPC: http://www.wowhead.com/npc=47296/helix-gearbreaker

Nie da się otworzyć lootu z bossa po zabiciu go. Drzwi nie otwierają się, przez co nie da się ukończyć instancji.

**Comment by Niriam** [comment]
I've witnessed it too, Helix doesn't even spawn after defeating Lumbering Oaf. I don't think he even rides his back like he should. Three times I was stuck on this moment and couldn't go forward.

**Comment by Nergleth** [comment]
Had the same problem a few minutes ago. There was no Helix, door never opened.

**Comment by hawcik** [comment]
Same problem here.

**Comment by mikisa** [comment]
Still not working

**Comment by Damfee** [comment]
I feel like this is an important bug, as you can't do more than half of the dungeon because of it...

**Comment by guthwine** [comment]
still not working

**Comment by Avalain** [comment]
Nope, no hope here. I just want to nag a bit and remind that's one of first dungeons new players see. When you hear "omg dis is bugged kik me plz" it's kinda discouraging.

Bug is not game breaking for some, but my world was shattered.
